HW1/P6/P6.py

```diff
@@ -0,0 +1,71 @@
+"""
+Written by Jaemin Cheun
+Harvard CS205
+Assignment 1
+October 6, 2015
+"""
+from random import randint
+import numpy as np
+import findspark
+findspark.init()
+
+from pyspark import SparkContext
+
+# initiaize Spark
+sc = SparkContext("local", appName="P6")
+sc.setLogLevel("ERROR")
+lines = sc.textFile("Shakespeare.txt")
+
+# filtering any words that do not satisfy the conditions in the problem
+def wordFilter(word):
+	return (not word.isdigit()) and (not word.isupper()) and (not (word.endswith('.') and word[:-1].isupper()))
```

Comment:
this might provide the correct filter? There are some exclamation points/other punctuations besides the "."